audit/projects/k8s-artifacts-prod/services/bigquery/bigquery.datasets.json

```diff
@@ -1,20 +0,0 @@
-[
-  {
-    "kind": "bigquery#dataset",
-    "id": "k8s-artifacts-prod:gcs_logs",
```

Comment:
I'm confused.  https://github.com/kubernetes/k8s.io/pull/2089 landed 13h ago.  It updated the audit script to dump these files, and preemptively included them from a manual run.

This is behaving like either:
- the audit script hasn't been updated
- the service account running this doesn't have sufficient bigquery read access to see these resources

Going to check log files

Reply:
https://github.com/kubernetes/k8s.io/pull/2100 should have fixed, but it broke the script's ability to run in CI

https://github.com/kubernetes/k8s.io/pull/2102 should fix those problems